spiderfoot/templates/opts.tmpl

```diff
@@ -140,7 +140,7 @@
                             instructions = ""
                             for step in apiKeyInstructions:
                                 if "https://" in step or "http://" in step:
-                                    step = re.sub(r'.*(https?://.[^ ]+).*', '<a target=_new href=\'\\1\'>\\1</a>', step)
+                                    step = re.sub(r'(https?://.[^ ]+)', '<a target=_new href=\'\\1\'>\\1</a>', step)
```

Comment:
This change seems reasonable. Submitting this as a separate PR would see the change landed sooner and prevent potential merge conflicts until this PR is reviewed.

As for putting configuration information in `apiKeyInstructions`, usually this would go in the description. Presumably other modules can/will integrate with local services in the future. If it is useful to keep configuration steps separate, it may be worth adding a separate module meta property for this.

Reply:
Thanks for your answer. Submitted as #1355.

I will update this PR to move the instruction to the module description. I also think a specific `dataSource` / `model` (currently set to `"FREE_AUTH_UNLIMITED"`) would be better. What do you think?